Feature(#423): Addition of Challenge 31 (third documentation challenge) (#757)
OWASP/wrongsecrets: src/main/resources/explanations/challenge31_hint.adoc

> We are looking for the secret in the `Web-page` of our GitHub repository. But how do we find it?
> 
> You can solve this challenge by the following steps:
> 
> 1. Go to our https://owasp.org/www-project-wrongsecrets/[Website].
> 2. Go through all the tabs and you can spot the Secret.

Reviewer: ```suggestion
You can solve this challenge by the following steps:

1. Go to our https://owasp.org/www-project-wrongsecrets/[Website].
2. Browse some tabs, surely the secret is in there :)
```
